dist/options/options.js

```diff
@@ -20,10 +20,12 @@ function saveOptions() {
     const exclude = document.getElementById('exclude-radio').checked;
     const labelList = document.getElementById('label-list');
     const labels = labelList.value.split(/[\n]+/).map(s => s.trim()).filter(s => !!s);
+    const bundleDates = document.getElementById('enable-radio').checked;
```

Comment:
Let's give the id a more specific name, maybe `group-by-date-checkbox` (see earlier comment about radio vs checkbox). 

In hindsight, the other ids/settings should have been given less vague names, but I think we can preemptively give better names to new settings to avoid clashes or confusion (while just leaving the old names as-is, as you mentioned).